common/app/common/dfp/HighMerchandiseComponentAgent.scala

```diff
@@ -0,0 +1,12 @@
+package common.dfp
+
+import model.Tag
+
+trait HighMerchandiseComponentAgent {
+
+  protected def highMerchandisingTargetedTags: HighMerchandisingTargetedTagSet
+
+  def hasHighMerchandiseTarget(tags: Seq[Tag]): Boolean = {
+    tags exists highMerchandisingTargetedTags.hasTag
```

Comment:
You might want to put a feature switch in here as well so that if the switch is off, this is always true.